### PR TITLE
Alicja Z. Portfolio Website

### PR DESCRIPTION
**Pull request by AlicjaZub**
Icons are still missing

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+				<nav>
+					<a class="current" href="#">Home</a>
+					<a href="about.html">About</a>
+					<a href="portfolio.html">Portfolio</a>
+					<a href="contact.html">Contact</a>
+					<a href="">CV</a>
+				</nav>
+			</div>
+			<div class="links">
+				<a  href="https://github.com/AlicjaZub" target="_blank"><img class="menu-image" src="githublight.png" alt="github logo"></a>
```
Comment (mporam) [inline index.html]: 2 spaces after the `<a` in the anchor tag and on the line below. should be 1

File: index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+				<a  href="https://www.linkedin.com/in/alicja-zubel-07a9515a/" target="_blank"><img class="menu-image" src="lin.png" alt="linkedin logo"></a>
+			</div>	
+		</section>
+		
+		<section class="right-section">
```
Comment (mporam) [inline index.html]: think a `<main>` tag would make more sense here as its the main page content

File: index.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+			<button class="hidden-menu-button">MENU</button>
+			<div class="info">
+				<div class="left">
```
Comment (mporam) [inline index.html:41]: try and avoid class names like "left" and "right" - there is usually something more semantic you can use

File: index.html [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+					<h1 class="welcome-heading">Welcome!</h1>
```
Comment (mporam) [inline index.html]: does this actually need a class? can you just target the h1 tag?

File: index.html [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+					<h2 class="welcome-heading2">My name is <span>Alicja</span> and I'm a something something</h2>
```
Comment (mporam) [inline index.html]: classes with numbers in like`welcome-heading2` can often be improved. this isn't the second heading, its the sub-heading. which makes a better class name

File: portfolio.html [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+			</div>
+		</section>
+		
+		<section class="right-section">  <!-- to be adjusted with projects -->
+			<button class="hidden-menu-button">MENU</button>
+			<div class="porfolio-heading">
+				<h1 class="welcome-heading" >Projects</h1>
+			</div>
+			<div class="info info-portfolio" >
+				<div class="left project-left">
```
Comment (mporam) [inline portfolio.html]: `project-left` is not very semantic, try using `project-img` then no class on the `` tag, and in the css you can use `.project-img img {}`

File: portfolio.html [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+					<img class="project-img" src="test.png" alt="project one">
+				</div>
+				<div class="right">
+					
+					<div class="project-details">
+						<h2>Project name</h2>
+						<p>Something about the project</p>
+						<p class="software-used-p">JavaScript &nbsp &nbsp PHP</p>
```
Comment (mporam) [inline portfolio.html]: `software-used-p` wont necessarily always be a `<p>` tag, why not just `software-used` ?

File: style.css [hunk 8]
```diff
@@ -0,0 +1,243 @@
...
+.container {
+	display: flex;
+	color: white;
+	font-size: 1rem;
+	font-family: 'Poppins', sans-serif;
+}
+
+p {
+	line-height: 1.5rem;
+	padding: 1%;
```
Comment (mporam) [inline style.css]: do all p tags really need 1% padding? seems a bit too general and might bite you later

File: style.css [hunk 9]
```diff
@@ -0,0 +1,243 @@
...
+	margin-right: 10%;
+}
+
+span {
+	color: white;
+}
+
+button {
+	border-radius: 100%; 
+	color: #ff7292;;
```
Comment (mporam) [inline style.css]: 2 colons at the end of the line

File: style.css [hunk 10]
```diff
@@ -0,0 +1,243 @@
...
+	background: #3b3837;
+	border-color: #ff7292;
+	cursor: pointer;	
+}
+
+.menu-image {
+	width: 2.2rem;
```
Comment (mporam) [inline style.css]: is there a reason you have used rems here instead of px or %?

File: style.css [hunk 11]
```diff
@@ -0,0 +1,243 @@
...
+
+.menu {
+	width: 15%;
+	display: flex;
+	flex-wrap: wrap;
+	border-right: solid black 1px;
+	justify-content: space-between;
+	position: fixed;
+	height: 100vh;
+	background-color: #3b3837;/*#393634*/
```
Comment (mporam) [inline style.css]: whats the comment for?

File: style.css [hunk 12]
```diff
@@ -0,0 +1,243 @@
...
+	padding-top: 30px;
+	padding-bottom: 30px;
+	border-bottom: solid 0.5px gray;
+	margin-left: 30%;
+	margin-right: 30%;
+	color: white;
+	
+}
+
+nav a:hover, nav a.current {
```
Comment (mporam) [inline style.css]: when comma seperating css we write it:
```
.selector1,
.selector2 {

}
```

File: style.css [hunk 13]
```diff
@@ -0,0 +1,243 @@
...
+
+nav a {
+	width: 100%;
+	padding-top: 30px;
+	padding-bottom: 30px;
+	border-bottom: solid 0.5px gray;
+	margin-left: 30%;
+	margin-right: 30%;
+	color: white;
+
```
Comment (mporam) [inline style.css]: empty line?

File: style.css [hunk 14]
```diff
@@ -0,0 +1,243 @@
...
+}
+
+nav a {
+	width: 100%;
+	padding-top: 30px;
+	padding-bottom: 30px;
+	border-bottom: solid 0.5px gray;
+	margin-left: 30%;
+	margin-right: 30%;
+	color: white;
```
Comment (mporam) [inline style.css]: text color instead of hex? its good to be consistent with color values, use all hex

File: style.css [hunk 15]
```diff
@@ -0,0 +1,243 @@
...
+	display: flex;
+	flex-wrap: wrap;
+	text-align: center;
+}
+
+nav a {
+	width: 100%;
+	padding-top: 30px;
+	padding-bottom: 30px;
+	border-bottom: solid 0.5px gray;
```
Comment (mporam) [inline style.css]: text color instead of hex, use hex when possible please

File: style.css [hunk 16]
```diff
@@ -0,0 +1,243 @@
...
+	margin-left: 30%;
+	margin-right: 30%;
+	color: white;
+	
+}
+
+nav a:hover, nav a.current {      
+    color: #ff7292;
+	border-color: #ff7292;
```
Comment (mporam) [inline style.css:101]: indentation is wrong here, the properties are not lined up

File: style.css [hunk 17]
```diff
@@ -0,0 +1,243 @@
...
+}
+	
+.navigation, .links, .logo {
+	width: 100%;
+}
+
+.links, .logo {
+	display: flex;
+	justify-content: space-around;
+
```
Comment (mporam) [inline style.css]: empty line

File: style.css [hunk 18]
```diff
@@ -0,0 +1,243 @@
...
+.hidden-menu-button {
+	position: absolute;
+	top: 2%;
+	right: 2%;
+	padding: 15px;
+	display: none;
+}
+
+.info {
+	display:flex;
```
Comment (mporam) [inline style.css]: missing space after the `:`

File: style.css [hunk 19]
```diff
@@ -0,0 +1,243 @@
...
+	justify-content: space-around;
+	padding: 30px;
+	margin: 7%;
+	height: auto;
```
Comment (mporam) [inline style.css]: generally `height: auto` does nothing, are you sure you need it?

File: style.css [hunk 20]
```diff
@@ -0,0 +1,243 @@
...
+.project-img { /*tbc depending on how the project looks*/
+	padding-top: 2%;
+	width: 100%;
+}
+
+.software-used-p {
+	color: #ff7292;
+	font-size: 0.8rem;
+	padding: 5%;
+
```
Comment (mporam) [inline style.css]: empty line?

File: style.css [hunk 21]
```diff
@@ -0,0 +1,243 @@
...
+.avatar { /*contact image can be as it is, tbc*/
+	width: 100%;
+}
+
+.contact, .left.contact-left {
+	text-align: center;
+	color: #ff7292;
+	width: 100%;
+}
```
Comment (mporam) [inline style.css:164]: missing line break after the `}`

File: style.css [hunk 22]
```diff
@@ -0,0 +1,243 @@
...
+
+.portfolio-img, .porfolio-heading .avatar {
+	width: 100%;
+	display: flex;
+	justify-content: center;
+}
+
+.porfolio-heading h1 {
+	margin-top: 6%;
+
```
Comment (mporam) [inline style.css]: empty line

File: style.css [hunk 23]
```diff
@@ -0,0 +1,243 @@
...
+	text-align: center;
+	border-top: solid 1px #ff7292;
+	margin-left: 10%;
+	margin-right: 10%;
+	margin-bottom: 10%;
+	padding: 10px;
+}
+
+.footer-p {
+	color: gray;
```
Comment (mporam) [inline style.css]: hex code please

File: style.css [hunk 24]
```diff
@@ -0,0 +1,243 @@
...
+footer {
+	text-align: center;
+	border-top: solid 1px #ff7292;
+	margin-left: 10%;
+	margin-right: 10%;
+	margin-bottom: 10%;
+	padding: 10px;
+}
+
+.footer-p {
```
Comment (mporam) [inline style.css]: why `.footer-p`? it wont necessarily always be a p tag. maybe `footer-text` might be better?

File: about.html [hunk 25]
```diff
@@ -0,0 +1,57 @@
...
+				<nav>
+					<a href="index.html">Home</a>
+					<a class="current"  href="about.html">About</a>
+					<a href="portfolio.html">Portfolio</a>
+					<a href="contact.html">Contact</a>
+					<a href="">CV</a>
+				</nav>
+			</div>
+			<div class="links">
+				<a  href="https://github.com/AlicjaZub" target="_blank"><img class="menu-image" src="githublight.png" alt="github logo"></a>
```
Comment (mporam) [inline about.html]: 2 spaces after the a before the href

File: contact.html [hunk 26]
```diff
@@ -0,0 +1,62 @@
...
+			</div>
+		</section>
+		
+		<section class="right-section">
+			<button class="hidden-menu-button">MENU</button>
+			<div class="info contact-info" >
+				<div class="contact">
+					<h1 class="welcome-heading">Contact details: </h1>
+				</div>
+				<div class="left contact-left">
```
Comment (mporam) [inline contact.html]: contact-left isnt a very semantic class name, try `contact-details`

**Review by mporam — CHANGES_REQUESTED** [review]
There is a vertical scroll on the home & about me page which seems unnecessary?

File: index.html (hunk 3 above)
Comment (AlicjaZub) [reply index.html:41]: We dicussed it and you agreed to keep it as it is

**Review by mporam — APPROVED** [review]

**Pull request merged**